library/src/main/java/com/owncloud/android/lib/resources/notifications/GetNotificationsRemoteOperation.java

```diff
@@ -66,39 +66,38 @@ public class GetNotificationsRemoteOperation extends RemoteOperation {
     private static final String NODE_DATA = "data";
 
     @Override
-    protected RemoteOperationResult run(OwnCloudClient client) {
-        RemoteOperationResult result = null;
-        int status = -1;
+    public RemoteOperationResult<List<Notification>> run(NextcloudClient client) {
+        RemoteOperationResult<List<Notification>> result;
+        int status;
         GetMethod get = null;
         List<Notification> notifications;
         String url = client.getBaseUri() + OCS_ROUTE_LIST_V12_AND_UP;
 
         // get the notifications
         try {
-            get = new GetMethod(url);
-            get.addRequestHeader(OCS_API_HEADER, OCS_API_HEADER_VALUE);
+            get = new GetMethod(url, true);
+            HashMap<String, String> map = new HashMap<>();
+            map.put("format", "json");
```

Comment:
`"format"` and `"json"` should be constants somewhere, they are also used in other operations.